presentations/traits/slides.adoc

```diff
@@ -34,30 +34,33 @@ include::./2.rs[]
 
 == Borrowing and Ownership of `self`
 
-It is not unlike normal, but at the beginning somewhat unfamiliar.
+It is like usual, but at the beginning somewhat unfamiliar.
```

Comment:
usual to what? I'm not sure what this sentence is trying to convey.

Reply:
Like Borrowing and Ownership of other values, I'll change that.

Reply:
```suggestion
It is like normal ownership and borrowing, but at the beginning somewhat unfamiliar.
```

Reply:
@jamesmunns New suggestion